chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -240,10 +254,14 @@ void BraveRenderViewContextMenu::ExecuteIPFSCommand(int id, int event_flags) {
 void BraveRenderViewContextMenu::ExecuteCommand(int id, int event_flags) {
   switch (id) {
     case IDC_COPY_CLEAN_LINK: {
-      auto link_url =
-          params_.link_url.is_valid()
-              ? params_.link_url
-              : GetSelectionNavigationURL(GetProfile(), params_.selection_text);
+      GURL link_url = params_.link_url;
+      if (!link_url.is_valid()) {
+        auto selected_url =
+            GetSelectedURL(GetProfile(), params_.selection_text);
+        if (selected_url.has_value()) {
+          link_url = selected_url.value();
+        }
```

Comment:
no need to handle `else` condition here?

Reply:
add return, but we cannot be here in that cases